util/site-dashboard/ot-nightly-results.js

```diff
@@ -11,7 +11,7 @@ const block_level_urls = {
   "csrng": ["https://reports.opentitan.org/hw/ip/csrng/dv/2024.01.22_10.48.06/"],
   "edn": ["https://reports.opentitan.org/hw/ip/edn/dv/2024.01.22_10.48.55/"],
   "entropy_src": ["https://reports.opentitan.org/hw/ip/entropy_src/dv/2024.01.22_10.49.35/"],
-  "flash_ctrl": ["https://reports.opentitan.org/hw/ip/flash_ctrl/dv/2024.01.22_10.51.11/"],
+  "flash_ctrl": ["https://reports.opentitan.org/hw/top_earlgrey/ip_autogen/flash_ctrl/dv/2024.01.22_10.51.11/"],
```

Comment:
This won't work, since that URL doesn't point anywhere. The date points to a regression run that happened before this PR merged. :)

On the side, we probably ought to go back to "latest" results, since regressions are running again.

Reply:
I don't know how to update the site proper, so I asked @hcallahan-lowrisc and @HU90m for help.

Reply:
I copied the January-dated run into the new directory layout, so that should get this PR to pass. (Can browse the public bucket [here](https://console.developers.google.com/storage/browser/reports.opentitan.org/) if interested).